src/lib/defaultCommands.js

```diff
@@ -494,15 +495,17 @@ var defaultCmds = {
 					var attrs  = '';
 
 					if (width) {
-						attrs += ' width="' + width + '"';
+						attrs += ' width="' + parseInt(width) + '"';
```

Comment:
`parseInt` should be using a base value, see [the MDN documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt) for more.

```suggestion
						attrs += ' width="' + parseInt(width, 10) + '"';
```